easybuild/easyconfigs/s/seqan/seqan-3.0.0.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = "Tarball"
+
+name = 'seqan'
```

Comment:
I think it should be called `SeqAn3`.
Using the lower case version would cause conflicts with the existing `SeqAn` configs.